services/frontend/src/components/ProfileCard/ProfileCard.js

```diff
@@ -0,0 +1,60 @@
+import {IconButton, Paper} from "@material-ui/core";
```

Comment:
QA notes (these must be fixed before approval):

- Background color doesn't extend over the whole page when scrolling. It should.
- Avatar on the `ProfileCard` isn't top-aligned with the person's name (as it is on the Profile page). It should be.
- The 'navigate to' arrow (i.e. the right chevron) inconsistently takes up more space on the card when compared with the `ProjectCard`. It should match the proportions.
- For some reason, the padding on the `Navbar` shrinks _only_ on the `People` tab. It shouldn't.
- The padding between the `PeopleHeader` and the `PeopleList` is inconsistent with the corresponding padding for the `ProjectsHeader` and the `ProjectsList` on the `Projects` tab. It shouldn't be.